### DISCUSSION TITLE
v from source android 7.1

### DISCUSSION THREAD

**Discussion by Nokiand** [vlang/v#15306] [Installation Issues]
most of the files appear but my version of clang does not have gc.h, gc.c; to use a mock up and allocate

**Comment by Nokiand** [comment]
Reviewed make and checked file structure cmd, doc, examples, thirdparty, tutorials, vc, vlib...  Since the cpu is arm v7a 32bit.  Header file <gc.h> was not found at the end of make and to ./v run hw.v builder error.  Where to install header?

**Reply by JalonSolov** [reply]
Wherever your C compiler normally looks for headers.  Otherwise you have to add extra command-line switches such as `-cflags="-I<directory where gc.h lives>"` - which may be different depending on your compiler.

**Comment by Nokiand** [comment]
android 7.1.1 termux 0.118.0 check V 0.3.1 not released yet ~/v $ ./v -nogc run hw.v : warning linker flags DT 0x8000001 unknown argument 'nogc' also ./v -autofree run hw.v : <gc.h> not found, needed for module 'builtin'.  Got gc.c & gc.h from | github.com/GJDuck/GC and made an object to link commenting out lines 263 & 294 'asm("movq %%rsp, %0" : "=r"(stack_ptr))' and 'asm("imul %2" : "=d"(z) "a"(x) "r"(y))'.  Make appears to use v_win.c not v.c, hope to re-compile, but is 64 bit code may only go one-way arm32 to A64, port?  First make.bat looks like dos, objdump : no such file, tcc.exe : does not work, make:***[GNU:makefile:100:all] Error 1 and linker warning 3 times with mcc -std=gnu99 -w -o v1.exe ./vc/v.c -lm -lpthread.

**Reply by JalonSolov** [reply]
The first message is correct.  There is no V 0.3.1 release.

**Reply by JalonSolov** [reply]
If you don't want gc, use `-gc none` instead of `-nogc`.  V has extensive help files to learn the correct command line options.

**Comment by Nokiand** [comment]
v -rwx 3330120 aug 6 ~/v $ head v ELF ~/v $ ./v version warn:flags V 0.3.0 dc78f1b ~/v $ ./v -gc none run h.v warn:flags warn:flags Hello World!  Also compiled and ran asm.v, hello_v_js.v looked at cmd/v/v.v -rwx 5260 jul 31, vc/v.c -rw 4935062 aug 6.  
~/v $ ./v -gc none symlink warn:flags warn:flags build error <gc.h> builtin.  Cannot find output file, must be interpreted, running, with missing parts, anyway.

**Reply by spytheman** [reply]
Do `export VFLAGS='-gc none'` before everything else, so that all compilations will get it.

**Comment by Nokiand** [comment]
Noticed Termux/Android instructions in readme.md, that V requires libexecinfo, libgc and libgc-static that appear with clang.
